macros/models/_sector/tokens/transfers_base.sql

```diff
@@ -0,0 +1,108 @@
+{% macro transfers_base(blockchain, traces, transactions, erc20_transfers, native_contract_address = null, wrapped_token_deposit = null, wrapped_token_withdrawal = null) %}
+{%- set token_standard_20 = 'bep20' if blockchain == 'bnb' else 'erc20' -%}
+{# denormalized tables are not yet in use #}
+{%- set denormalized = True if blockchain in ['base'] else False -%}
+
+WITH transfers AS (
+    SELECT block_time
+    , block_number
+    , tx_hash
+    , NULL AS evt_index
+    , trace_address
+    {% if native_contract_address%}
+    , {{native_contract_address}} AS contract_address
+    {% else %}
+    , CAST(NULL AS varbinary) AS contract_address
+    {% endif %}
+    , 'native' AS token_standard
+    , "from"
+    , to
+    , value AS amount_raw
+    FROM {{ traces }}
+    WHERE success
+    AND (call_type NOT IN ('delegatecall', 'callcode', 'staticcall') OR call_type IS null)
+    AND value > UINT256 '0'
+    {% if is_incremental() %}
+    AND {{incremental_predicate('block_time')}}
+    {% endif %}
+
```

Comment:
Are you planning to add gas fees further in spells? It's also sort of transfer which affects balances

Reply:
Do you have any more insight on this? gas fees makes balances a bit tricky but aren't a lot of the fees in the traces?

Reply:
you can do just:

`select "from" as spender, gas_price * gas_used as amount from {blockchain}.transactions`

but it's better to doublecheck if this is applicable for L2 as well

Reply:
Thank you. I will look into this in a follow up PR.